external/js/kyber/publish.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Need better comments in here to explain what's so weird about this package that requires it has a special way of publishing it.